Userland/Applications/Spreadsheet/Spreadsheet.cpp

```diff
@@ -82,24 +82,30 @@ size_t Sheet::add_row()
     return m_rows++;
 }
 
-static size_t convert_from_string(StringView str, unsigned base = 26, StringView map = {})
+static constexpr size_t convert_from_string(StringView str)
```

Comment:
Not sure why you're getting rid of the `map` argument and the behaviour related to it?

Reply:
With generic function (`base` and `map` as argument), we have to use `find()` to lookup character in `map`.

As Spreadsheet will always use 26 characters ('A' to 'Z'), we can use this knowledge to implement this function
in more optimal way.
1. Instead of find, perform ASCII substraction 'A' to get `digit_value`.
2. `base` is hard coded as 26. 

I will keep old behaviour, if we want to have more flexibility and generic nature.